reporters_db/data/laws.json

```diff
@@ -1587,6 +1638,23 @@
             "variations": []
         }
     ],
+    "Hein's": [
+        {
+            "cite_type": "treaty",
+            "end": null,
+            "examples": [
+                "Hein's No. KAV 2053"
+            ],
+            "href": "https://law.resource.org/pub/us/code/blue/IndigoBook.html#T6",
+            "jurisdiction": "United States",
+            "name": "Hein’s United States Treaties and Other International Agreements",
+            "regexes": [
+                "$reporter\\s*(?:No\\.|Number)?\\s*KAV\\s*(?P<number>\\d+)"
```

Comment:
What is KAV here.

Reply:
There is something called Kavass's Current Treaty Index and Hein's treaty reporter organizes treaties by their Kavass numbers, hence KAV.